features/spaces/invitations.feature

```diff
@@ -39,13 +39,18 @@ Feature: Spaces: Invitations
     And that Invitation is Accepted
     And all other Invitations to that Invitation's contact info are Resolved
 
-  @unstarted @andromeda
+  @built @andromeda
   Scenario: Accepting an Invitation as a Guest
-    Given an Invitation was sent less than 14 days ago
-    When a Guest accepts that Invitation
-    Then the Guest becomes a Space Member
-    And that Invitation is Accepted
-    And all other Invitations to that Invitation's contact info are Resolved
+    Given a "System Test" Space
+    And the "System Test" Space has a Space Owner "space-owner@example.com"
+    And an Invitation to the "System Test" Space is sent by Space Owner "space-owner@example.com"
+      | name | email                       |
+      | A Guest | a-guest@example.com |
+    When the Invitation to "a-guest@example.com" for the "System Test" Space is accepted by the Guest "a-guest@example.com"
+    Then the Guest "a-guest@example.com" becomes a Space Member of the "System Test" Space
+    And the Invitation to "a-guest@example.com" for the "System Test" Space has a status of "accepted"
+    # @todo implement me!
```

Comment:
Is there a more "proper" Cucumber annotation that could be used here, instead of a comment? That way we would get reminders in the Cucumber output when we run the tests.

Reply:
It looks like we could have two options:

1. We could mark the step as `pending,` which would fail the test, and let us know we didn't implement this step yet.
2. We could make the step a no-op, where it would just "pass," but that would obscure the fact that it's not quite there yet.

I think I'm leaning towards leaving it as "@todo" in the code; so that someone who notices it and has the capacity can go attempt to implement it. 

That said, we don't really have norms or guidance around how to handle `@todo.` I tend to think of them as an 'aspirational message for future Zee' and not a "someone else cleans this up because I don't give any fucks." Still, I anticipate that others may have different reads on what `@todo` means).

Reply:
(Note: We also have a checkbox for finishing this) https://github.com/zinc-collective/convene/issues/117

Reply:
I like trying to align on `{@}todo` as meaning "someone who notices it and has the capacity can go attempt to implement it". We could also document this intent in the `CONTRIBUTING` guide or something, if we have the capacity for that.

Reply:
Yea, I need to run atm but would happily merge someone adding it!